cartridges/notify_me/cartridge/models/product/decorators/notifyAttributes.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+var URLUtils = require("dw/web/URLUtils");
+
+module.exports = function (object, apiProduct) {
```

Comment:
A redundant second argument. You don't pass it when using the decorator.
```suggestion
decorators.notifyAttributes(product);
```